Firestore/src/FieldValue/DocumentTransformTrait.php

```diff
@@ -30,9 +30,9 @@ trait DocumentTransformTrait
     private $args;
 
     /**
-     * @param array $args
+     * @param mixed $args
```

Comment:
Please update the other docblocks to `mixed` as well

Reply:
Thanks! Could we also please hit the class variable declaration as well?

Reply:
ack, sorry i missed that. done now.